torch/_inductor/utils.py

```diff
@@ -1144,6 +1144,13 @@ def blue_text(msg):
     return _color_text(msg, "blue")
 
 
+def _max_clock_rate(msg):
+    import pynvml
```

Comment:
Is this a new dependency?  Will it be installed for all OSS users?

Reply:
It is an optional dependency today that is only necessary for some specific monitoring tools (like temperature or process running on the GPU) https://github.com/pytorch/pytorch/blob/1128cf96f078a76f27b038d7adbc2b72fe8927e2/torch/cuda/__init__.py#L53C1-L53C12
We should not assume it is there indeed.

Reply:
Ah, this is unfortunate - it's used by torch.cuda.clock_rate() and several other helps, so I assumed it was fair game, but it turns out it's not, and there's kind of an annoying history here.  Sigh.  Gonna just live with this failing internally.